Tests/Tests/OptionalSpec.swift

```diff
@@ -91,13 +91,37 @@ class OptionalSpec: XCTestCase {
         // (m >>= f) >>= g = m >>= (\x -> f x >>= g)
         property("associativity law") <- forAll { (o: OptionalOf<Int>, fa: ArrowOf<Int, Int>, fb: ArrowOf<Int, Int>) in
             let m = o.getOptional
-            let f: Int -> Int? = compose(pure, fa.getArrow)
-            let g: Int -> Int? = compose(pure, fb.getArrow)
+            let f: Int -> Int? = pure • fa.getArrow
+            let g: Int -> Int? = pure • fb.getArrow
 
             let lhs = (m >>- f) >>- g
             let rhs = m >>- { x in f(x) >>- g }
 
             return lhs == rhs
         }
+
+        // (f >=> g) >=> h = f >=> (g >=> h)
+        property("left-to-right Kleisli composition of monads") <- forAll { (x: Int, fa: ArrowOf<Int, Int>, fb: ArrowOf<Int, Int>, fc: ArrowOf<Int, Int>) in
```

Comment:
Line Length Violation: Line should be 100 characters or less: currently 156 characters (line_length)<br>Variable Name Min Length Rule Violation: Variable name should be 2 characters or more: currently 1 characters (variable_name_min_length)<br>Variable Name Min Length Rule Violation: Variable name should be 3 characters or more: currently 2 characters (variable_name_min_length)